Main.java

```diff
@@ -0,0 +1,722 @@
+
+    public String getNume() {
+        return nume;
+    }
+    public void setNume(String nume) {
+        this.nume = nume;
+    }
+
+    public Carte[] getCarti() {
+        return carti;
+    }
+    public void setCarti(Carte[] carti) {
+        this.carti = carti;
+    }
+
+    Sectiune(String nume, Carte[] carti){
+        this.nume = nume;
+        this.carti = carti;
+    }
+
+    @Override
+    public String toString() {
+        return "Sectiune{" +
+                "nume='" + nume + '\'' +
+                ", carti=" + Arrays.toString(carti) +
+                '}';
+    }
+}
+
+class Angajat {
+    private String nume;
+    private String prenume;
+    public static final int SALARIU_BAZA = 2000;
+
+    // Getters and Setters
+
+    public String getNume() {
+        return nume;
+    }
+    public void setNume(String nume) {
+        this.nume = nume;
+    }
+
+    public String getPrenume() {
+        return prenume;
+    }
+    public void setPrenume(String prenume) {
+        this.prenume = prenume;
+    }
+
+    public int getSalariu(){
+        return this.SALARIU_BAZA;
+    }
+
+    Angajat(String nume, String prenume){
+        this.nume = nume;
+        this.prenume = prenume;
+    }
+
+    @Override
+    public String toString() {
+        return "Angajat{" +
+                "nume='" + nume + '\'' +
+                ", prenume='" + prenume + '\'' +
+                ", salariu=" + SALARIU_BAZA +
+                '}';
+    }
+}
+
+class Bodyguard extends Angajat{
+    private String firmaSecuritate;
+
+    Bodyguard(String nume, String prenume, String firmaSecuritate) {
+        super(nume, prenume);
+        this.firmaSecuritate = firmaSecuritate;
+    }
+
+    public int getSalariu() {
+        return this.SALARIU_BAZA + 1500;
+    }
+
+    // Getters and Setters
+
+    public String getFirmaSecuritate() {
+        return firmaSecuritate;
+    }
+    public void setFirmaSecuritate(String firmaSecuritate) {
+        this.firmaSecuritate = firmaSecuritate;
+    }
+
+    @Override
+    public String toString() {
+        return "Bodyguard" + "\n" +
+                "firmaSecuritate: " + firmaSecuritate;
+    }
+}
+
+class Bibliotecar extends Angajat{
+    // grad = I/II/III/debutant
+    private String grad;
+
+    Bibliotecar(String nume, String prenume, String grad) {
+        super(nume, prenume);
+        this.grad = grad;
+    }
+
+    public int getSalariu() {
+        return this.SALARIU_BAZA + 2000;
+    }
+
+    // Getters and Setters
+
+    public String getGrad() {
+        return grad;
+    }
+    public void setGrad(String grad) {
+        this.grad = grad;
+    }
+
+    @Override
+    public String toString() {
+        return "Bibliotecar" + "\n" +
+                "grad: " + grad;
+    }
+}
+
+interface Interfata{
+    void vizualizareAng(Angajat[] ang);
+
+    Angajat[] adaugaAng(Angajat[] ang, Angajat a);
+
+    Angajat[] citireDateAngajatNou(Scanner scanner, Angajat[] ang, ClasaServiciu cs);

```

Comment:
since the interface is implemented in ClasaServiciu, why do you need a parameter for ClasaServiciu? in the implementation of the method in ClasaServiciu you can call ClasaServiciu's methods without the need to have a param at this method